astropy/io/votable/tree.py

```diff
@@ -1149,7 +1152,10 @@ def __init__(self, votable, ID=None, name=None, datatype=None,
                  xtype=None,
                  config=None, pos=None, **extra):
         if config is None:
-            config = {}
+            if hasattr(votable, '_get_version_flags'):
```

Comment:
Is it possible to pass in an old table?

Reply:
I do not know if it is possible to pass in an old table, but without this change two tests fail because the correct unit format could not be determined.

Reply:
OK, sounds like it is necessary, then, though I'll leave this to @tomdonaldson just to be sure.

Reply:
I was wondering about this check as well.  I think it is OK since It seems necessary to protect the unit tests which are exercising `Field` without a containing `votable`.  Testing `Field` that way (or designing `Field` that way) may be questionable, but addressing those questions would be outside the scope of this PR.